### DISCUSSION TITLE
Deciding the purpose of the Hirob

### DISCUSSION THREAD

**Discussion by Maraza25** [Ubn-Jr/hirob-mo-main#11] [Ideas]
Dear teammates, we need to make some important decisions together within the Hirob robot project. We have to make general decisions for the progress of the project and the quality of the final product.

**Here are the main topics we need to decide:**

* Hirob Project Overview
* Movement Mechanisms 
* Robot Design
* Map
* Mobile Application Features

**Sample questions related to these topics**

* Should Hirob be controlled via the keypad on the robot or via a mobile app?
* Does Hirob need tracks or wheels for its movement mechanism?
* Should the map be dynamically generated through the mobile application, or should it have a ready-to-use map system within the application?
* Should Hirob's outer container be made of plastic or wood?
* Should the map be dynamic or static puzzle-like format?
* What kind of design should Hirob's outer appearance have?

**Your opinions and suggestions on these topics are extremely valuable for our project. Also, please feel free to share with us any new ideas or questions that come to your mind. I am waiting for your ideas to make it better together.**

**Comment by Maraza25** [comment]
### My Thoughts on Hirob's Design:

**Control Mechanism:** I envision controlling Hirob through both a mobile app and an external remote control or website. Given the difficulties some students in schools have accessing phones, I believe this flexibility is crucial.

**Movement Mechanism:** I am really fascinated by the movement mechanism of the emo robot. I would like to incorporate this system into Hirob if possible.

**Map Selection:** It would be nice to have both a pre-designed map and the ability for users to create their own maps.

For other aspects of Hirob's design, I am still thinking and have not come to a definite decision.

**Comment by arzuuddemir** [comment]
Hello, I would like to answer a few questions asked above. 
- If the HIROB robot proceeds with the keypad, it provides a better advantage than the mobile application. It is both fast to use and simple. But the mobile application is a disadvantage for those who do not have a phone, tablet, etc.
- I think the HIROB robot should move with wheels. Because both its design and mechanism are easy and it moves better when it has 4 wheels.
- It would be better to use a dynamic map. Because it has many good features and is reliable in terms of the robustness of the project.
- Plastic is better for durability, but wood can also be used for a natural look and durability.
- Static puzzles are better for children. Mathematical terms and the logic of the algorithm using a computer sit very well in their heads.
- Honestly, I don't have a good idea about this question. But here are some of the things that come to my mind: Use more vivid colors and make it look more interesting. Facial features can be placed on the answer to the question (frowning when angry, smiling when the correct answer is found). I will continue to share as I think of them.

**Comment by FurkanTansel** [comment]
Hello.
1-hi-rob should be made of wood to avoid health problems 
2- mobile background colors should be compatible with wood colors
3- controlling the hi-rob from mobile is a good idea, but it would be costly to have a remote control and the remote control cannot be wooden anyway
4- It is a good idea for the user to build the map from the mobile application, but it would be good to offer a few maps as a mobile team in advance.
